src/test/java/edu/kis/powp/jobs2d/CommandUsageCounterVisitor.java

```diff
@@ -0,0 +1,24 @@
+package edu.kis.powp.jobs2d;
+
+import edu.kis.powp.jobs2d.command.*;
+import edu.kis.powp.jobs2d.command.visitor.CommandVisitorInterface;
+
+public class CommandUsageCounterVisitor implements CommandVisitorInterface {
+    public int operateToCounter = 0;
+    public int setPositionCounter = 0;
```

Comment:
These fields should be private and accessed only using getters (so counting wouldn't be corrupted from outside)